runtime/basilisk/src/weights/xcmp_queue.rs

```diff
@@ -66,4 +66,11 @@ impl<T: frame_system::Config> WeightInfo for BasiliskWeight<T> {
 			.saturating_add(T::DbWeight::get().reads(1 as u64))
 			.saturating_add(T::DbWeight::get().writes(1 as u64))
 	}
+
+	fn service_deferred() -> Weight {
+		Weight::default()
```

Comment:
benchmarks needed

Reply:
just merged rate limiter stuff into master that has benchmarks for this